src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
+/**
+ * Handles fetching and saving user data.
+ */
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet {
+
+    private Datastore datastore;
+
+    @Override
+    public void init() {
+        datastore = new Datastore();
+    }
+
+    /**
+     * Responds with the "about me" section for a particular user.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+
+        String user = request.getParameter("user");
```

Comment:
Do I understand correctly that a single logged-in user is allowed to view the "about me" page of a number of different users? If so, I found this a little confusing in that I was assuming in a lot of the code below that "user" was referring to the currently logged-in user, when it is actually referring to the user whose "about me" page is being viewed. You might want to add a comment and give a name to distinguish between the two different users in question (the logged-in user vs the page subject).

Reply:
I agree, it might be better to have clarification on what user we are talking about in line 38.